tests/unit_node/_fs/_fs_appendFile_test.ts

```diff
@@ -64,6 +64,9 @@ Deno.test({
 
 Deno.test({
   name: "Async: Data is written to passed in rid",
+  // TODO(bartlomieju): this test is broken in Deno 2, because `file.rid` is undefined.
```

Comment:
I'm concerned about these TODOs being lost and not taken care of before the release. Maybe we should label them like `TODO(2.0)`?

Reply:
@dsherret this is not really a Deno 2 issue. These APIs are just outright broken and have always been. They're covered in https://github.com/denoland/deno/issues/23012.

Reply:
I'll link back that PR in the issue.

Reply:
But the test was previously passing and now it's not?

Reply:
I wasn't thinking. This seems find to me.